### PR TITLE
🐛 Fix amp-access login

### LINKED ISSUE #2526: Login button does not work
https://amp.dev/documentation/examples/e-commerce/checkout_flow/preview/?format=websites

### PR DESCRIPTION
**Pull request by lluerich**
Since the ```/login.html``` in the ```/examples/static/```-folder was not reachable, I moved it to the same folder as the API itself.

@sebastianbenz I hope this is ok. If you come up with a better solution, let me know. 😃

Fixes #2526

### CODE REVIEW

File: examples/api/amp-access/amp-access.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ function handleAuthorization(request, response) {
 
 function handleLogin(request, response) {
   const returnUrl = request.query.return;
-  const filePath = utils.project.absolute('/examples/static/samples/files/login.html');
+  const filePath = utils.project.absolute('/examples/api/amp-access/login.html');
```
Comment (sebastianbenz) [inline examples/api/amp-access/amp-access.js]: you can use relative paths instead `./login.html`.

File: examples/api/amp-access/amp-access.js [hunk 2]
```diff
@@ -38,10 +38,10 @@ const POWER_USERS = {
 };
 const EXPIRATION_DATE = 24*60*60*1000; // 1 day in ms
```
Comment (sebastianbenz) [inline examples/api/amp-access/amp-access.js]: please rename this file from amp-access.js to index.js.

File: examples/api/amp-access/amp-access.js (hunk 1 above)
Comment (lluerich) [reply examples/api/amp-access/amp-access.js]: Do you mean by using ```utils.project.relative(...``` or just the string ```./login.html```?

File: examples/api/amp-access/amp-access.js (hunk 2 above)
Comment (lluerich) [reply examples/api/amp-access/amp-access.js]: Thanks for pointing that out. PTAL.

File: examples/api/amp-access/amp-access.js (hunk 1 above)
Comment (lluerich) [reply examples/api/amp-access/amp-access.js]: ```./login.html``` only works, if you set up ```nunjucks.configure(__dirname)``` beforehand.
Was this what you were thinking of?
Either way, I will move the ```filePath``` into a constant.

Comment (sebastianbenz) [reply examples/api/amp-access/amp-access.js]: ah ok, `path.join(__dirname, 'login.html')` would've been my next suggestion.

Comment (lluerich) [reply examples/api/amp-access/amp-access.js]: works for me, and 1 less line of code 👍

**Pull request merged**